x/erc20/keeper/ibc_callbacks.go

```diff
@@ -101,15 +100,25 @@ func (k Keeper) OnRecvPacket(
 		return ack
 	}
 
-	pair, _ := k.GetTokenPair(ctx, pairID)
-	if !pair.Enabled {
+	pair, found := k.GetTokenPair(ctx, pairID)
```

Comment:
@zale144 won't it fail on `GetTokenPairID` before if not registered?

Reply:
not necessarily fail, more like return early. I guess it would, even though those are separate stores. the extra `found` check is probably redundant.